### PR TITLE
Improve LTI automatic syncing

### PR DESCRIPTION
**Pull request by andrew-gardener**
- added logging to lti grade pushback (to help debug potential grade issues)
- moved `assignAllVisibleSetsToUser` that I added to `CourseUpdater.updateUser` to `LTIBridge._updateLaunchUser`. This might have been further slowing down lti membership ext syncs (though from my testing on staging it wasn't having any affect). by moving it into _updateLaunchUser we can still ensure that a user will be assigned all active assignments when they enter webwork via lti launch without performing this action for every user on membership update.
- removed `()` from lti context variable fetching in updateclass_lti.pl (might cause errors when a variable is empty)
- added single quotes around varables in `autoupdate.pl` (in case course name, consumer key, or context id ever contain anything that might be problematic for the command line)

### CODE REVIEW

File: lib/WebworkBridge/Importer/CourseUpdater.pm [hunk 1]
```diff
@@ -215,12 +215,6 @@ sub updateUser
 			$self->addlog("Teaching staff $id rejoined $courseid");
 		}
 	}
-
-	# assign all visible homeworks to students
```
Comment (xcompass) [inline lib/WebworkBridge/Importer/CourseUpdater.pm]: why is this removed?

Comment (andrew-gardener) [reply lib/WebworkBridge/Importer/CourseUpdater.pm]: This is something I added at the beginning of the refactor to ensure that launch users would be assigned to all the homework assignments (with the byproduct of it being added to membership extension. wasn't part of membership originally). 

I decided to instead moved it to https://github.com/ubc/webwork2/pull/10/files#diff-8d3c16b0cea6523366dc0f312870b006R597 which will still assign all available homework on launch but will no longer do it during membership (the concern being the extra amount of time it takes for the membership extension to run, especially for large classes)
